src/Employer.Web/Controllers/CreateVacancyController.cs

```diff
@@ -25,14 +24,14 @@ public IActionResult Index()
         }
 
         [HttpPost("new-vacancy", Name = RouteNames.NewVacancy_Index_Post)]
-        public async Task<IActionResult> Index(IndexViewModel vm)
+        public async Task<IActionResult> Index(string employerAccountId, IndexViewModel vm)
```

Comment:
For the vacancyId work I've done I've just added it as a property to the viewmodel and the Model binder automatically populates it from the route.  I like only having one parameter being posted to an action - less ambiguous.

Reply:
Will have a think about this but sounds good.